napari/layers/vectors/vectors.py

```diff
@@ -371,11 +358,11 @@ def feature_defaults(self):
 
         See `features` for more details on the type of this property.
         """
-        return self._feature_defaults
+        return self._feature_manager.defaults
 
     @property
     def property_choices(self) -> Dict[str, np.ndarray]:
-        return _features_to_choices(self._features)
+        return self._feature_manager.choices()
```

Comment:
are those calling to deprecated properties/choice of feature manger, and if that's the case shoudl they be deprecated as well ?

Reply:
See comment: https://github.com/napari/napari/pull/3904/files#r782328666